Sources/FileSystem/Configuration/TestflightConfigurationProcessor.swift

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+import Model
+
+public struct TestflightConfigurationProcessor: ResourceWriter {
+
+    let path: ResourcePath
+
+    public init(path: ResourcePath) {
+        self.path = path
+    }
+
+    public func writeConfiguration(
+        apps: [Model.App],
+        testers: [Model.BetaTester],
+        groups: [Model.BetaGroup]
+    ) {
+        // TODO: Convert models to a TestflightConfiguration and write it to disk
```

Comment:
if we do the Convert in FileSystem Model. Where do we do the comparison of configs? 🤔

Reply:
My feel at the moment is that we're going to convert the config into models and then we can compare the models somewhere else to produce the actions